tools/integration_tests/util/operations/file_operations.go

```diff
@@ -574,6 +595,16 @@ func SyncFile(fh *os.File, t *testing.T) {
 	}
 }
 
+func SyncFileShouldThrowStaleHandleError(fh *os.File, t *testing.T) {
```

Comment:
Please don't create test helpers for the code under test: https://google.github.io/styleguide/go/best-practices.html#leave-testing-to-the-test-function